numpy/linalg/linalg.py

```diff
@@ -2179,6 +2221,10 @@ def lstsq(a, b, rcond="warn"):
     LinAlgError
         If computation does not converge.
 
+    See Also
+    --------
+    scipy.linalg.lstsq : Compute least-squares solution to equation Ax = b.
```

Comment:
Would it be better to use a description like "equivalent function in scipy", especially in cases where there's no obvious difference?

It's a little confusing to see a docstring like:
```
Do X

See Also
---------
scipy.x : Do X
```

Reply:
This comment applies to all the functions with identical names in the two modules, correct?  I like the suggestion, but I'll use "Similar function..." instead of "Equivalent function...".  Even in cases where the function is currently a drop-in replacement, it might not be in the future.

Reply:
Correct, this was a general comment dropped by random chance on this particular function. "Similar" is indeed a better word.

Reply:
I made the change in my latest update.